source/Contribution-Guide/addingpages.html.md.erb

```diff
@@ -1,25 +1,27 @@
 ---
 title: Adding Pages
-last_reviewed_on: 2024-01-26
+last_reviewed_on: 2025-02-25
 review_in: 12 months
 weight: 10
 ---
 
 # <%= current_page.data.title %>
 
-These pages are written in the Markdown, lightweight markup language and stored with a github repo.
+These pages are written in Markdown, a lightweight markup language, and stored in a GitHub repository.
 
 ## Prerequisites
 
-- Basic knowledge of using markdown to create documents
+- Basic knowledge of using markdown to create documents - see [Markdown Guide](https://www.markdownguide.org/) / [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/)
 - Access to the repo [hmcts/ops-runbooks](https://github.com/hmcts/ops-runbooks)
 
-## Documentation
+## Documentation Process
 
-Initially, documents should be created in OneNote, Word, or other feature rich editor, to enable the information to be captured.  Once the document is ready, save each page as a file within the folder using the extension .html.md.erb 
+1. Initially create documents in a feature-rich editor (e.g., OneNote, Word) to capture information
+2. Once the document is ready, save each page as a file with the extension ``.html.md.erb`` in the appropriate folder
+
+## Index Page Structure
+The first page in every folder should be named `index.html.md.erb` and start with these 7 lines of code:
```

Comment:
`start with these 7 lines of code:` Do we want to say this, people will copy the heading. `Contribution Guide` blindly instead of creating their own.